ui/Screen/SendFunds.svelte

```diff
@@ -0,0 +1,256 @@
+
+  .submit {
+    display: flex;
+    justify-content: flex-end;
+    padding-top: 1.5rem;
+  }
+
+  .from-to {
+    display: grid;
+    grid-template-columns: 13rem 1.5rem 13rem;
+    grid-column-gap: 1rem;
+    margin-top: 1rem;
+  }
+
+  .from {
+    display: flex;
+    justify-content: flex-end;
+  }
+</style>
+
+<ModalLayout dataCy="page">
+  <div class="wrapper">
+    {#if step === 1}
+      <header>
+        <div class="icon">
+          <Icon.ArrowUp style="fill: var(--color-primary)" />
+        </div>
+        <Title variant="big">Outgoing transfer</Title>
+      </header>
+      <Title
+        style="color: var(--color-foreground-level-6); padding: 0 0.5rem 0.5rem
+        0.5rem;">
+        To
+      </Title>
+      <Input.Text
+        bind:value={toAddress}
+        placeholder="Enter recipient address"
+        style="flex: 1; padding-bottom: 0.5rem;" />
+      <Title style="color: var(--color-foreground-level-6); padding: 0.5rem;">
+        Amount
+      </Title>
+      <Input.Text
+        placeholder="Enter the amount"
+        bind:value={amount}
+        showLeftItem
+        style="flex: 1; padding-bottom: 0.5rem;">
+        <div slot="left" style="display: flex;">
+          <Icon.Currency
+            size="big"
+            style="fill: var(--color-foreground-level-6)" />
+        </div>
+      </Input.Text>
+      <Title style="color: var(--color-foreground-level-6); padding: 0.5rem;">
+        From
+      </Title>
+      <!-- TODO(julien): shouldn't identity.accountId be the same as fromAddress -->
+      <Dropdown
+        placeholder="Select wallet you want to use"
+        bind:value={fromAddress}
+        options={dropdownOptions} />
+      <div class="submit">
+        <Button on:click={() => nextStep()}>Review transfer</Button>
+      </div>
+    {/if}
+    {#if step === 2}
+      <header>
+        <div class="icon">
+          <Icon.ArrowUp style="fill: var(--color-primary)" />
+        </div>
+        <Title variant="big">Outgoing transfer</Title>
+        <div class="from-to">
+          <div class="from">
+            {#each dropdownOptions as option}
+              {#if option.value === fromAddress}
+                <Avatar
+                  size="small"
+                  title={option.avatarProps.title}
+                  variant={option.avatarProps.variant}
+                  avatarFallback={option.avatarProps.avatarFallback} />
+              {/if}
+            {/each}
+          </div>
+          <Icon.ArrowRight />
+          <div class="to">
+            <Title truncate style="color: var(--color-foreground-level-6);">
+              {toAddress}
+            </Title>
+          </div>
+        </div>
+      </header>
+      <Row variant="top" style="">
+        <div slot="left">
+          <Text
+            variant="regular"
+            style="color:var(--color-foreground-level-6);">
+            Amount
+          </Text>
+        </div>
+
+        <div slot="right">
+          <Rad rad={amount} />
```

Comment:
```suggestion
          <Rad rad={$amountStore} />
```

Reply:
Will this work if the amount get's changed in the form?

Reply:
Yeah it's reactive so if the component is subscribed to it, it should update automatically. The `$` prefix indicates the subscription. Happy to look through this and push an example commit if that would be helpful 😊

Reply:
Nice one. Wondering if we should rename the variables. https://github.com/radicle-dev/radicle-upstream/pull/712/commits/059b135f36003ee31ca3b3d725e96732a61fe2bc

Reply:
addressed https://github.com/radicle-dev/radicle-upstream/pull/712/commits/059b135f36003ee31ca3b3d725e96732a61fe2bc